pkgs/applications/misc/epdfview/default.nix

```diff
@@ -4,11 +4,12 @@ stdenv.mkDerivation rec {
   name = "epdfview-0.1.8";
 
   src = fetchurl {
-    url = "http://trac.emma-soft.com/epdfview/chrome/site/releases/${name}.tar.bz2";
-    sha256 = "1w7qybh8ssl4dffi5qfajq8mndw7ipsd92vkim03nywxgjp4i1ll";
+    url = "http://http.debian.net/debian/pool/main/e/epdfview/epdfview_0.1.8.orig.tar.gz";
```

Comment:
Consider using `mirror://debian` instead. Also, consider factoring out the version.